cmd/ebrelayer/contract/contract.go

```diff
@@ -0,0 +1,31 @@
+package contract
+
+// -------------------------------------------------------
+//    Contract
+//
+//		Contains functionality related to the smart contract
+// -------------------------------------------------------
+
+import (
+	"io/ioutil"
+	"log"
+	"strings"
+
+	"github.com/ethereum/go-ethereum/accounts/abi"
+)
+
+func LoadABI() abi.ABI {
+	// Open the file containing Peggy contract's ABI
+	rawContractAbi, errorMsg := ioutil.ReadFile("cmd/ebrelayer/contract/PeggyABI.json")
```

Comment:
the path should be a constant

Reply:
fixing in next push

Reply:
https://github.com/cosmos/peggy/commit/b9b756658265b22958b404a7a09eb289ac1ede77